code/game/objects/structures/crates_lockers/crates/graves.dm

```diff
@@ -0,0 +1,116 @@
+/obj/structure/closet/crate/grave
+	name = "burial mound"
+	desc = "A marked patch of soil, adorned with a wooden cross"
+	icon_state = "grave"
+	dense_when_open = TRUE
+	material_drop = /obj/item/stack/ore/glass/basalt
+	material_drop_amount = 5
+	opened = TRUE
+	anchorable = FALSE
+	anchored = TRUE
+	locked = TRUE
+	breakout_time = 900
+	cutting_tool = TOOL_SHOVEL
+	var/lead_tomb = FALSE
+	var/first_open = FALSE
+
+/obj/structure/closet/crate/grave/attackby(obj/item/W, mob/user, params)
+	.=..()
+	if(istype(W, /obj/item/screwdriver))
+		if(!user.is_literate())
+			to_chat(user, "<span class='notice'>You scratch illegibly on [src]!</span>")
+			return
+		var/t = stripped_input(user, "What would you like the inscription to be?", name, null, 53)
+		if(user.get_active_held_item() != W)
+			return
+		if(!user.canUseTopic(src, BE_CLOSE))
+			return
+		if(t)
+			desc = "[t]"
+		return
+
+/obj/structure/closet/crate/grave/open(mob/living/user, obj/item/S, force = FALSE)
+	if(!opened)
+		to_chat(user, "<span class='notice'>The ground here is too hard to dig up with your bare hands. You'll need a shovel.</span>")
+	else
+		to_chat(user, "<span class='notice'>The grave has already been dug up.</span>")
+
+/obj/structure/closet/crate/grave/tool_interact(obj/item/S, mob/living/carbon/user)
+	if(user.a_intent == INTENT_HELP) //checks to attempt to dig the grave, must be done on help intent only.
+		if(!opened)
+			if(S.tool_behaviour == cutting_tool)
+				to_chat(user, "<span class='notice'>You start start to dig open \the [src]  with \the [S]...</span>")
+				if (do_after(user,20, target = src))
+					opened = TRUE
+					locked = TRUE
+					dump_contents()
+					update_appearance()
+					SEND_SIGNAL(user, COMSIG_ADD_MOOD_EVENT, "graverobbing", /datum/mood_event/graverobbing)
+					if(lead_tomb == TRUE && first_open == TRUE)
```

Comment:
I should have been more specific but you should remove this part that gives people the permanent trauma. It should only have been in the ruin so it has no purpose anymore.